.github/workflows/ci.yml

```diff
@@ -66,30 +66,13 @@ jobs:
         run: gcc test/test-stack.c -o a.out && ./a.out
 
       - name: Generate parser from scratch and test it
-        if: ${{ runner.os == 'Linux' }}
+        if: ${{ runner.os == 'Linux' || needs.changedfiles.outputs.c }}
         shell: bash
         run: |
           npm install
           npm run build
           npm test
 
-      - name: Check parser
```

Comment:
This change would be removed after https://github.com/tree-sitter/tree-sitter-scala/pull/287 resolves